sys-libs/libspeexdsp.py

```diff
@@ -0,0 +1,30 @@
+import stdlib
+from stdlib.template import autotools
+from stdlib.manifest import manifest
+
+
+@manifest(
+    name='libspeexdsp',
+    category='sys-libs',
+    description='''
+    Speex is an Open Source/Free Software patent-free audio compression format designed for speech.
+    ''',
+    tags=['sound', 'speech', 'libs', 'gnu'],
```

Comment:
```suggestion
    tags=['sound', 'speech', 'audio', 'media'],
```

Reply:
Why remove the `gnu` tag ? [Speex](https://www.speex.org/) is part of the GNU project

Reply:
Couldn't find it in savanaah but looks like you're correct according to their home page.

Reply:
Also https://www.gnu.org/software/
(https://www.gnu.org/software/speex/ redirects to https://www.speex.org/)